changelog.md

```diff
@@ -30,6 +30,10 @@
 
 - The `{.this.}` pragma, deprecated since 0.19, has been removed.
 - `nil` is no longer a valid value for distinct pointer types.
+- The `type Foo = object {.final.}` pragma syntax for types, which was
+  deprecated since Nim 0.20.0, has been removed.
+- Removed two type pragma syntaxes deprecated since 0.20, namely
+  `type Foo = object {.final.}`, and `type Foo {.final.} [T] = object`.
```

Comment:
```suggestion
- Removed two type pragma syntaxes deprecated since 0.20, namely
  `type Foo = object {.final.}`, and `type Foo {.final.} [T] = object`.
```
The first one is just a duplicate, isn't it?

Reply:
Must have been a rebase issue due to the different merge rules in changelog